common/build-style/python3-pep517.sh

```diff
@@ -33,9 +33,9 @@ do_install() {
 		make_install_target="${wheelbase//-/_}-${version}-*-*-*.whl"
 	fi
 
-	# If do_build was overridden, make sure the TMPDIR exists
+	# If do_build was overridden, make sure the TMP exists
 	mkdir -p build
-	TMPDIR=build python3 -m pip install --use-pep517 --prefix /usr \
+	TMP=build python3 -m pip install --use-pep517 --prefix /usr \
```

Comment:
```suggestion
	TMPDIR="${PWD}/build" python3 -m pip install --use-pep517 --prefix /usr \
```
As above, an absolute path seems to fix the build.